showmore_dyn/templates/settings.tpl

```diff
@@ -0,0 +1,12 @@
+<div class="settings-block">
+        <h3 class="settings-heading">{{$title}}</h3>
```

Comment:
Code standards: Please use a consistent indenting, either tabs or spaces but it should be the same in the whole file.